.golangci.yml

```diff
@@ -0,0 +1,262 @@
+# This file contains all available configuration options
```

Comment:
They different in the details, i.e. the sorting order for import paths is a different for every repository. It might also make sense to disable depreciation warnings in some repos for the whole codebase instead of just for a few files (e.g. the `api` repository).

So while it would be helpful if we could share that file, I'm a bit hesitant since there are some differences.

Reply:
Extra disabled warnings I understand (as temporary until we clean up). But things like import sorting order - that's exactly what I think should be identical in every repo.

Reply:
The sorting order is:

```golang
import (
	// std library packages
	"log"

	// everything else
	"google.golang.org/grpc"
	"github.com/spacemeshos/merkle-tree"

	// packages from the same repo
	"github.com/spacemeshos/poet/server"
	"github.com/spacemeshos/poet/signal"
)
```

So unless we want to mix packages from the same repo into the everything else category - which I'm opposed to - we have to use a different sorting configuration for every repo.

Reply:
Right! 😭